tor_ocr/main.py

```diff
@@ -40,8 +41,12 @@
 
 # "helloworld" is a valid API key, however use it sparingly
 __OCR_API_KEY__ = os.getenv('OCR_API_KEY', 'helloworld')
-# __OCR_API_URL__ = 'https://api.ocr.space/parse/image'  # free API url
-__OCR_API_URL__ = 'https://apipro1.ocr.space/parse/image'
+# __OCR_API_URLS__ = ['https://api.ocr.space/parse/image']  # free API url
+__OCR_API_URLS__ = [
+    'https://apipro1.ocr.space/parse/image',  # USA
+    'https://apipro2.ocr.space/parse/image',  # Europe
+    'https://apipro3.ocr.space/parse/image'  # Asia
```

Comment:
Maybe add a space before the comment, so all three comments are aligned?

Reply:
I just saw your nits, but official response: PEP-8 calls for two spaces before the # in an inline comment, which is why those are not even. I did think about that, though. :D